custom/enikshay/nikshay_datamigration/factory.py

```diff
@@ -63,11 +63,13 @@ def person(self):
                 # 'owner_id': self._location.location_id,
                 'update': {
                     'aadhaar_number': self.patient_detail.paadharno,
+                    'age': self.patient_detail.page,
                     'age_entered': self.patient_detail.page,
                     'contact_phone_number': validate_number(self.patient_detail.pmob),
                     'current_address': self.patient_detail.paddress,
                     'current_address_district_choice': self.patient_detail.Dtocode,
                     'current_address_state_choice': self.patient_detail.scode,
+                    'dob_known': False,
```

Comment:
> The other booleans are all set by the migration (not the app)

I see. I think it would still be good to map `True` and `False` to `('true', 'false') ` (or `('yes', 'no')`) because those are the conventions we've been using in the app. 

> Question: since dob never gets loaded in the app, is it needed? I suppose it might be used in reports or elsewhere.

Looks to me like `dob` is used in [some places](https://india.commcarehq.org/a/enikshay-test/apps/view/e6c88dc7cf8c742cbf397eff07d9a245/modules-7/forms-0/source/). Take a look at how the "age" hidden value is calculated there, and how "dob" is fetched. (I'm not saying this is good...)

I'd be wary of going off the form summary for figuring out where case properties are used as we have a lot of casedb lookups that are happening in the app. I've been unzipping and grepping through the app build files to find references to things if I'm not sure.